src/app/auth/PageLogin.tsx

```diff
@@ -21,6 +23,26 @@ export const PageLogin = () => {
         <Box p="6" borderRadius="md">
           <LoginForm onSuccess={onLogin} />
         </Box>
+        <Box>
+        Discover {' '}
+          <Link
+            color="violet"
```

Comment:
Use the brand colors. 

maybe 200 or 300

```suggestion
            color="brand.200"
```